packages/daimo-common/src/currencies.ts

```diff
@@ -16,7 +16,8 @@ export const currencyRateUSD: CurrencyExchangeRate = {
 
 const data: [string, string, string, number][] = [
   ["Euro", "€", "EUR", 2],
-  ["Argentine Peso", "A$", "ARS", 0],
+  ["Argentine Peso", "ARS", "ARS", 0],
```

Comment:
Did you mean to change the Argentinian Peso symbol to ARS or did you want to keep it A$ 

google says it can be both but A$ might save space in input row